cmsl1t/utils/module.py

```diff
@@ -17,3 +18,12 @@ def exists(module_name):
         return hasattr(m, tokens[-1])
     else:
         return True
+
+
+def load_L1TNTupleLibrary(lib_name='L1TAnalysisDataformats.so'):
+    import ROOT
+    PROJECT_ROOT = os.environ.get('PROJECT_ROOT', os.getcwd())
+    if lib_name not in ROOT.gSystem.GetLibraries():
+        path_to_lib = os.path.join(PROJECT_ROOT, 'build', lib_name)
+        ROOT.gSystem.Load(path_to_lib)
+        assert (lib_name in ROOT.gSystem.GetLibraries())
```

Comment:
I find `assert`s typically quite frustrating since if they fail there's little to no documentation about why it shouldn't be doing this.  How about replacing with `if lib_name not in ROOT.gSystem.GetLibraries():`, an appropriate error message and then raise some exception?

Reply:
OK, can do.

Reply:
Also, just realised that ROOT will also complain:
```
Error in <TUnixSystem::FindDynamicLibrary>: /code/build/sdsd[.so | .dll | .dylib | .sl | .dl | .a] does not exist in .:/software/cmsl1t/miniconda/envs/cms/lib:
```

which also reminds me: We need to add the build directory to the `LD_LIBRARY_PATH`

Reply:
done